mne/io/proj.py

```diff
@@ -801,8 +805,10 @@ def make_eeg_average_ref_proj(info, activate=True, verbose=None):
                            'one EEG reference to another.')
 
     logger.info("Adding average EEG reference projection.")
-    eeg_sel = pick_types(info, meg=False, eeg=True, ref_meg=False,
-                         exclude='bads')
+
+    ch_dict = {**{type_: True for type_ in ch_type},
+               'meg': False, 'ref_meg': False}
+    eeg_sel = pick_types(info, **ch_dict, exclude='bads')
```

Comment:
I think you can reuse the private function `_picks_to_idx`

Reply:
Thanks @agramfort. I don't think I understand this private function though. How can I include it?

`eeg_sel =  _picks_to_idx(info, ch_type, none='data', exclude='bads', allow_empty=False, with_ref_meg=False)`???

Reply:
I would go for `none='eeg'`